pubsublite/create_topic.go

```diff
@@ -0,0 +1,55 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package pslite
+
+import (
+	"context"
+	"fmt"
+	"io"
+
+	"cloud.google.com/go/pubsublite"
+)
+
+// [START pubsublite_create_topic]
+
+func createTopic(w io.Writer, projectID, region, zone, topicID string) error {
+	// projectID := "my-project-id"
+	// region := "us-central1"
+	// zone := "us-central1-a"
+	// topicID := "my-topic"
+	ctx := context.Background()
+	client, err := pubsublite.NewAdminClient(ctx, region)
+	if err != nil {
+		return fmt.Errorf("pubsublite.NewAdminClient: %v", err)
+	}
+	defer client.Close()
+
+	const gib = 1 << 30
+	topic, err := client.CreateTopic(ctx, pubsublite.TopicConfig{
+		Name:                       fmt.Sprintf("projects/%s/locations/%s/topics/%s", projectID, zone, topicID),
+		PartitionCount:             2,        // Must be >= 1 and cannot decrease after creation.
+		PublishCapacityMiBPerSec:   4,        // Must be >= 4 and <= 16.
+		SubscribeCapacityMiBPerSec: 8,        // Must be >= 4 and <= 32.
+		PerPartitionBytes:          30 * gib, // Must be 30 GiB-10 TiB.
```

Comment:
Not sure if we should specify a max storage of 10 TiB, as it's not a hard limit unlike the others. It's the default API quota limit, but it can be increased upon request. 

Perhaps just say Must be >= 30 GiB? And optionally make a note to request higher API quota if limits are reached.

Same for the update topic example.